local-dev/templates/rawls.conf

```diff
@@ -93,8 +93,8 @@ dataRepo {
 
 resourceBuffer {
   projectPool {
-    regular = "cwb_ws_dev_v7"
-    exfiltrationControlled = "vpc_sc_v10"
+    regular = "cwb_ws_dev_v8"
```

Comment:
@jgainerdewar I updated these pools in the local dev config to reflect the new pools you recently created because the old values were giving me trouble when trying to run Rawls locally. Do these new values seem correct? (The `exfiltrationControlled` value I was particularly unsure of.)

Reply:
Yup, those look correct!